src/pages/cli/graphql/search-and-result-aggregations.mdx

```diff
@@ -160,11 +160,13 @@ query MyQuery {
 }
 ```
 
-In the example above, the response's `total` field contains the total search result count for "Students" whose name ends with "Brandel". 
+In the example above, the response's `total` field contains the total search result count for "Students" whose name ends with "Brandel".
+This total will be calculated based on all records, in the case of pagination even if not all records can be returned at once the total will still be representative of all matching results.
```

Comment:
do we need to explicitly call this out? Isn't this the intuitive answer? I recognize we didn't do this behavior in the past, therefore needed a note. Maybe more concise:

```suggestion
Note: `total` is calculated based on all records, irrespective of pagination configurations.
```

Reply:
It can be intuitive for some, but for someone without experience I could easily see an argument being made for either. I could imagine someone thinking their result is for the records they have in hand, and then using it as something it isn't